### DISCUSSION TITLE
entering edit mode with foreign language keyboard(Korean)

### DISCUSSION THREAD

**Discussion by amidiot** [Comcast/react-data-grid#2783] [Q&A]
I'm not able to write foreign language(Korean, in my case) into cell instantly.

If I type a letter when cell is selected, the first letter always becomes English letter,
then Korean letters are typed.

![image](https://user-images.githubusercontent.com/17634307/149157300-5af99d38-407d-4b1f-8ca7-734e664ed647.png)

How do I write Korean right into the cell when I type it?
Is there any locale settings?

**Comment by nstepien** [comment]
See https://github.com/adazzle/react-data-grid/issues/1702
IIRC we need to somehow detect IME input from outside a text box.
